openfish-webapp/src/utils/keypoints.ts

```diff
@@ -0,0 +1,40 @@
+import { BoundingBox, type Keypoint } from '../api/annotation'
+
+// Keypoints must be sorted.
+export function findClosestKeypointPair(
+  keypoints: Keypoint[],
+  currentTime: number
+): [Keypoint, Keypoint] {
+  for (let i = 0; i < keypoints.length - 1; i++) {
+    const kp = keypoints[i]
+    const kpNext = keypoints[i + 1]
+
+    if (kp.time === currentTime) {
+      return [kp, kp]
+    }
+    if (kpNext.time > currentTime) {
+      return [kp, kpNext]
+    }
+  }
+
+  throw new Error('Current time outside range of keypoints')
```

Comment:
Let's keep javascript error messages starting with a lowercase like we do in go. That way it's just more consistent.